src/components/AttachmentPicker/index.native.js

```diff
@@ -129,8 +140,11 @@ class AttachmentPicker extends Component {
             return;
         }
 
-        const result = getDataForUpload(attachment);
-        this.completeAttachmentSelection(result);
+        getDataForUpload(attachment).then((result) => {
+            this.completeAttachmentSelection(result);
+        }).catch(() => {
```

Comment:
Question here: Do we want to accept a possible `error`, then do `throw error` like `showDocumentPicker` does?